app/controllers/JobsController.scala

```diff
@@ -246,6 +246,29 @@ class JobsController @Inject()(jobDAO: JobDAO,
       } yield Ok(js)
     }
 
+  def runComputeMeshFileJob(organizationName: String,
+                            dataSetName: String,
+                            layerName: String,
+                            mag: String,
+                            agglomerateView: Option[String]): Action[AnyContent] =
+    sil.SecuredAction.async { implicit request =>
+      for {
+        organization <- organizationDAO.findOneByName(organizationName) ?~> Messages("organization.notFound",
+                                                                                     organizationName)
+        _ <- bool2Fox(request.identity._organization == organization._id) ?~> "job.export.notAllowed.organization" ~> FORBIDDEN
```

Comment:
`job.export.notAllowed.organization` message is currently limited to tiff export. Maybe word the message differently or create a new one.